src/sql_query.ts

```diff
@@ -32,8 +33,12 @@ export default class SqlQuery {
     }
 
     replace(options, adhocFilters) {
+        const view = {
+            METRIC_FIRST_BYTE: true
```

Comment:
do you propose replace this hardcoded view to some dynamic context? which context?

Reply:
@Slach  indeed this won't be hardcoded. I put just to have a test passing and verifying that mustache works. More updated to come in the following days.